speech-to-speech/README.md

```diff
@@ -0,0 +1,88 @@
+# Cloud Speech to Speech Translation gRPC Swift Sample
+
+This app demonstrates how to make gRPC connections to the(API Integration):
```

Comment:
`This app demonstrates how to make gRPC connections to the [Cloud Speech API](https://cloud.google.com/speech/)`